textract2page/cli.py

```
@@ -14,19 +14,26 @@
     type=click.Path(dir_okay=False, writable=True, exists=False, allow_dash=True),
 )
 @click.argument("aws-json-file", type=click.Path(dir_okay=False, exists=True))
-@click.argument("image-file", type=click.Path(dir_okay=False, exists=True))
-def cli(output_file, aws_json_file, image_file):
+@click.argument("image-file", type=str)
```

Please note that with this change to the `image-file` argument, a path that does not exist will not be caught here. I'd welcome ideas about the best way to preserve path checking while allowing the user to supply a filename when the image is not present. Perhaps a 2nd parameter would be the best way, perhaps in the form of a dedicated command that doesn't mess with the original one? Or is this ok, and we can throw an error in the convert_file function if the referenced file can't be found? (Again, please forgive my lack of Python skills and familiarity with Python idioms.)

---

Yes, interdependencies are not easily covered by Click. (It's doable via [Cloup](https://cloup.readthedocs.io/en/stable/), but not necessary.)

I would rather keep the `type`, but merely set `exists=False` to skip verification of the path, and then cover all combinations in a simple assertion, see below.

---

@bertsky Makes sense!